### PR TITLE
Update and organize provided graphs

### PR DESCRIPTION
**Pull request by Hwurzburg**
have not done Copter yet, but EKF,Plane, and QuadPlane

### CODE REVIEW

File: MAVProxy/tools/graphs/ekf3Graphs.xml [hunk 1]
```diff
@@ -1,272 +1,272 @@
...
 
-  <graph name='EKF3/Accel Bias Lane1'>
+  <graph name='EKF3/Solution Values/Accel Bias Lane1'>
     <description>EKF3 Accel Bias Lane1</description>
     <expression>XKF2[0].AX XKF2[0].AY XKF2[0].AZ</expression>
     <expression>XKF2.AX XKF2.AY XKF2.AZ</expression>
   </graph>
 
-  <graph name='EKF3/Accel Bias Lane2'>
+  <graph name='EKF3/Solution Values/Accel Bias Lane2'>
```
Comment (tridge) [inline MAVProxy/tools/graphs/ekf3Graphs.xml:137]: I'd prefer /State/ instead of /Solution Values/

File: MAVProxy/tools/graphs/mavgraphs2.xml [hunk 2]
```diff
@@ -354,44 +349,335 @@
...
+  <description></description>
+  <expression>MAG2.MagX-expected_mag_yaw(GPS,ATT,MAG2).x MAG2.MagY-expected_mag_yaw(GPS,ATT,MAG2).y MAG2.MagZ-expected_mag_yaw(GPS,ATT,MAG2).z</expression>
+ </graph>
+
+ <graph name='Copter/Tuning Ch6'>
+  <description></description>
+  <expression>PTUN.TunVal PTUN.Param:2</expression>
+ </graph>
+
+ <graph name='EKF3/Innovations/State Variances/EK3 Variances 1-11'>
```
Comment (tridge) [inline MAVProxy/tools/graphs/mavgraphs2.xml:643]: a variance isn't an innovation, maybe EKF3/Variances/ ?

**Review by tridge — CHANGES_REQUESTED** [review]

File: MAVProxy/tools/graphs/ekf3Graphs.xml (hunk 1 above)
Comment (Hwurzburg) [reply MAVProxy/tools/graphs/ekf3Graphs.xml]: done

File: MAVProxy/tools/graphs/mavgraphs2.xml (hunk 2 above)
Comment (Hwurzburg) [reply MAVProxy/tools/graphs/mavgraphs2.xml]: done

**Review by tridge — APPROVED** [review]

**Pull request merged**